wiki/BBCode/en.md

```diff
@@ -199,14 +199,24 @@ If specified through their user ID, the text between the open and closed tags wi
 [/list]
 ```
 
-The `[list]` tag is used to automatically format numerous types of lists throughout the osu! forums by using an asterisk enclosed in brackets (`[*]`) to indicate a new item in the list (shown above). By default, this will create a plain, bulleted list.
+The `[list]` tag is used to automatically format two different types of lists throughout the osu! forums by using an asterisk enclosed in brackets (`[*]`) to indicate a new item in the list (shown above). By default, this will create a plain, bulleted list.
 
-Other list styles can be formatted by specifying the `TYPE` argument as `1`, `a`, `A`, `i`, or `I`, which will format lists as numbered, lettered (lowercase), lettered (uppercase), roman numeral (lowercase), and roman numeral (uppercase) respectively.
+If the `TYPE` argument is specified as `1`, the list will be numbered.
```

Comment:
It can also be numbered if a value other than "1" is specified

Reply:
Made it similar to 202's end.

> If the `TYPE` argument is specified as `1`, it will create a numbered list.

Reply:
As mentioned in the description, I wanted the article to be understandable more easily.